next.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
 
   i18n,
   images: {
-    domains: ['storage.googleapis.com', 'i.ytimg.com', 'googleusercontent.com', 'localhost'],
+    domains: ['storage.googleapis.com', 'i.ytimg.com', 'googleusercontent.com', 'localhost', 'gateway'],
```

Comment:
This wasn't necessary, right? If I recall correctly, I think the issue here was actually solved by configuring the docker-compose correctly. Will rever this change, we can add it back if necessary